backend/db/dao.go

```diff
@@ -62,6 +62,15 @@ func CreateUser(request model.User, db *gorm.DB) error {
 	return nil
 }
 
+func UpdatePass(request model.PassUpdate, db *gorm.DB) error {
+	var user model.User
+	result := db.Save(request) //will need to test this
```

Comment:
have we tested this yet?

Reply:
No, maybe tomorrow we can talk about how to test it. Im unaware of the exact process for our current testing methods.